packages/runtime/runtime-utils/src/dataStoreHelpers.ts

```diff
@@ -73,6 +73,7 @@ export function responseToException(response: IResponse, request: IRequest): Err
 
 /**
  * Takes a set of packages and joins them pkg1/pkg2... etc. Tags the field as a code artifact
+ * @deprecated - use tagCodeArtifacts instead
```

Comment:
I'm curious how you're keeping track of all these deprecations in recent PRs to remove them in the next `next` branch. Don't want them to get forgotten.

Reply:
i just remove them as soon as i see the pr in next.

Reply:
speaking of deprecations, can we remove TaggedLoggerAdapter?

Reply:
> i just remove them as soon as i see the pr in next.

Last I talked with Sasha, we're supposed to let deprecations ship in a major before removing in the following major.

Reply:
> speaking of deprecations, can we remove TaggedLoggerAdapter?

Yes good catch!  We moved LTS for loader up past 0.45 a while back I believe.